docker/hoodie/hadoop/datanode/Dockerfile

```diff
@@ -17,7 +17,7 @@
 
 ARG HADOOP_VERSION=2.8.4 
 ARG HADOOP_DN_PORT=50075
-FROM apachehudi/hudi-hadoop_${HADOOP_VERSION}-base:latest
+FROM apachehudi/hudi-hadoop_${HADOOP_VERSION}-base:bullseye
```

Comment:
Similar here in `Dockerfile`s: the image tag will be reverted from `bullseye` to `latest` once the PR is close to merging.